exercises/clock/src/clock.h

```diff
@@ -0,0 +1,9 @@
+#ifndef _CLOCK_H
+#define _CLOCK_H
+
+#define MAX_STR_LEN 5
+
+void clock(char time_text[MAX_STR_LEN], int hour, int minute);
```

Comment:
Would it make sense to typedef `char time_text[MAX_STR_LEN]` to `clock_t` or similar?